.circleci/config.yml

```diff
@@ -42,40 +42,9 @@ jobs:
     environment:
       GO111MODULE: "on"
 
```

Comment:
Why cut out all the sanity checks?

Reply:
@edwarnicke there are 2 separate jobs `sanity-check` and `build`. Apparently, the latter is never referenced and looks like a stale historical piece just laying around. I propose to remove it.